[2025-03-26T00:47:50.691+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: titanic_survival_classifier.prepare_data_task manual__2025-03-26T00:42:43.687439+00:00 [queued]>
[2025-03-26T00:47:50.707+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: titanic_survival_classifier.prepare_data_task manual__2025-03-26T00:42:43.687439+00:00 [queued]>
[2025-03-26T00:47:50.708+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-03-26T00:47:50.749+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): prepare_data_task> on 2025-03-26 00:42:43.687439+00:00
[2025-03-26T00:47:50.764+0000] {standard_task_runner.py:60} INFO - Started process 1613 to run task
[2025-03-26T00:47:50.769+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'titanic_survival_classifier', 'prepare_data_task', 'manual__2025-03-26T00:42:43.687439+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpnc4_8m6r']
[2025-03-26T00:47:50.771+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask prepare_data_task
[2025-03-26T00:47:50.798+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-26T00:47:50.862+0000] {task_command.py:423} INFO - Running <TaskInstance: titanic_survival_classifier.prepare_data_task manual__2025-03-26T00:42:43.687439+00:00 [running]> on host 0f3b2e2ddf0a
[2025-03-26T00:47:51.000+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='titanic_survival_classifier' AIRFLOW_CTX_TASK_ID='prepare_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T00:42:43.687439+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T00:42:43.687439+00:00'
[2025-03-26T00:47:51.003+0000] {logging_mixin.py:188} INFO - ---Preparing train data---
[2025-03-26T00:47:51.034+0000] {logging_mixin.py:188} INFO - {'Schema Validation': 'No Issue with Schema', 'Missing Values': {'Age': '19.87%', 'Cabin': '77.10%', 'Embarked': '0.22%'}, 'Duplicate Rows': 0, 'Duplicate IDs': 0}
[2025-03-26T00:47:51.047+0000] {logging_mixin.py:188} INFO - Imputed Age (numerical) with median: 28.0
[2025-03-26T00:47:51.050+0000] {logging_mixin.py:188} INFO - Imputed Embarked (categorical) with mode: S
[2025-03-26T00:47:51.065+0000] {logging_mixin.py:188} INFO - Saved processed train data to /opt/***/data/train/processed_train.pkl
[2025-03-26T00:47:51.066+0000] {logging_mixin.py:188} INFO - ---Preparing test data---
[2025-03-26T00:47:51.081+0000] {logging_mixin.py:188} INFO - {'Schema Validation': 'No Issue with Schema', 'Missing Values': {'Age': '20.57%', 'Fare': '0.24%', 'Cabin': '78.23%'}, 'Duplicate Rows': 0, 'Duplicate IDs': 0}
[2025-03-26T00:47:51.094+0000] {logging_mixin.py:188} INFO - Imputed Age (numerical) with median: 27.0
[2025-03-26T00:47:51.096+0000] {logging_mixin.py:188} INFO - Imputed Fare (numerical) with median: 14.4542
[2025-03-26T00:47:51.109+0000] {logging_mixin.py:188} INFO - Saved processed test data to /opt/***/data/test/processed_test.pkl
[2025-03-26T00:47:51.110+0000] {python.py:201} INFO - Done. Returned value was: {'train_path': '/opt/***/data/train', 'test_path': '/opt/***/data/test'}
[2025-03-26T00:47:51.153+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=titanic_survival_classifier, task_id=prepare_data_task, execution_date=20250326T004243, start_date=20250326T004750, end_date=20250326T004751
[2025-03-26T00:47:51.229+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-26T00:47:51.282+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
